aimon/_utils/_utils.py

```diff
@@ -30,6 +30,14 @@
 CallableT = TypeVar("CallableT", bound=Callable[..., Any])
 
 
+static_system_prompt = "    Please generate a new response, and take into consideration the following factors:\
```

Comment:
I would move this from this autogenerated file to its own file

Reply:
Moved to resources->reprompt.py